fdbserver/storageserver.actor.cpp

```diff
@@ -810,9 +813,17 @@ struct StorageServer {
 		// and the lengths of both parameters.
 		Counter mutationBytes;
 
+		// Similar to mutationBytes, it counts the size of the serialized key and value for each mutation.
+		Counter logicalInputBytes;
+
+		// Bytes fetched from TLogs for moving-in shard.
+		Counter logicalMoveOverheadBytes;
+
 		Counter sampledBytesCleared;
 		// The number of key-value pairs fetched by fetchKeys()
 		Counter kvFetched;
+		// The throuput of the storage engine.
+		RateCounter kvThroughput;
```

Comment:
using Counter to counter the bytes of data to the storage engine, the counter will have three output "rate roughness total". The rate will be the throughput.